### PR TITLE
Update long-running.md

### PR DESCRIPTION
**Pull request by noydavidi**
Adding information about port assignment

## Status
Ready
## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-12259)

### CODE REVIEW

**Comment by kobymeir** [comment]
# Preview Site Available

Congratulations! The automatic build has completed successfully.
A preview site is available at: https://xsoar-pan-dev--pull-request-1669-buaunwvx.web.app

---
**Important:** Make sure to inspect your changes at the preview site.

Detected modified urls:
*  [Long Running Containers](https://xsoar-pan-dev--pull-request-1669-buaunwvx.web.app/docs/integrations/long-running)

**Comment by kobymeir** [comment]
# Preview Site Available

Congratulations! The automatic build has completed successfully.
A preview site is available at: https://xsoar-pan-dev--pull-request-1669-buaunwvx.web.app

---
**Important:** Make sure to inspect your changes at the preview site.

Detected modified urls:
*  [Long Running Containers](https://xsoar-pan-dev--pull-request-1669-buaunwvx.web.app/docs/integrations/long-running)

File: docs/integrations/long-running.md [hunk 1]
```diff
@@ -26,6 +26,27 @@ You can distinguish it from the rest of the containers by its name:
...
+- TAXII2 Server
+- TAXIIServer
+
+In these cases, the long-running behavior is inherent and cannot be modified.
+These integrations can operate in either "single engine" mode or "no engine" mode.
+
+**Single Engine:** 
+- Select your engine in the integration's configuration.
+- When the engine is selected, a new field called "Listen Port" will appear.
+- You must specify the port number that the instance will use.
```
Comment (ShirleyDenkberg) [inline docs/integrations/long-running.md:43]:
```suggestion
1. Select your engine in the integration's configuration.
2. When the engine is selected, a new field called "Listen Port" will appear.
3. Specify the port number that the instance will use.
```

File: docs/integrations/long-running.md [hunk 2]
```diff
@@ -26,6 +26,27 @@ You can distinguish it from the rest of the containers by its name:
...
+
+**No Engine:**
+- Select "no engine" in the integration configuration.
+- Click on "Save" and the server automatically assigns a port for the instance.
```
Comment (ShirleyDenkberg) [inline docs/integrations/long-running.md:47]:
```suggestion
1. Select "no engine" in the integration configuration.
2. Click **Save** and the server automatically assigns a port for the instance.
```

File: docs/integrations/long-running.md [hunk 3]
```diff
@@ -26,6 +26,27 @@ You can distinguish it from the rest of the containers by its name:
...
+Please note: If you select "No Engine", the listen port is assigned only after saving the configuration.
```
Comment (ShirleyDenkberg) [inline docs/integrations/long-running.md:48]:
```suggestion
Note: If you select **No Engine**, the listen port is assigned only after saving the configuration.
```

File: docs/integrations/long-running.md [hunk 4]
```diff
@@ -26,6 +26,27 @@ You can distinguish it from the rest of the containers by its name:
...
+You must click "Save" before clicking "Test" or performing any other actions with the instance.
```
Comment (ShirleyDenkberg) [inline docs/integrations/long-running.md:49]:
```suggestion
You must click **Save** before clicking **Test** or performing any other actions with the instance.
```

**Comment by ShirleyDenkberg** [comment]
@noydavidi Doc review completed.

**Review by Shellyber — APPROVED** [review]
Nice!

File: docs/integrations/long-running.md [hunk 5]
```diff
@@ -26,6 +26,28 @@ You can distinguish it from the rest of the containers by its name:
 
 ![image](/doc_imgs/integrations/66122754-2d337b00-e5e9-11e9-8775-562e228e3fe6.png)
 
+Certain integrations are long-running by default, they do not include the "Long Running Instance" checkbox. Examples of such integrations include:
+
+- Generic Export Indicators Service
```
Comment (Shellyber) [inline docs/integrations/long-running.md:31]:
```suggestion
- Generic Export Indicators Service (EDL)
```

File: docs/integrations/long-running.md [hunk 6]
```diff
@@ -26,6 +26,28 @@ You can distinguish it from the rest of the containers by its name:
...
+These integrations can operate in either "single engine" mode or "no engine" mode.
+
+**Single Engine:** 
+1. Select your engine in the integration configuration.
+2. When the engine is selected, a new field called "Listen Port" will appear.
+3. Specify the port number that the instance will use.
+
+**No Engine:**
+1. Select "no engine" in the integration configuration.
+2. Click **Save** and the server automatically assigns a port for the instance.
```
Comment (Shellyber) [inline docs/integrations/long-running.md:47]: So this behavior is for the 8.8 server version right?
In 8.9, it is no longer the case; they can just test regularly.
So let's add a disclaimer for this, regarding the server versions

**Review by Shellyber — COMMENTED** [review]
Also after the changes, please ask Shirley to review again :)

File: docs/integrations/long-running.md [hunk 7]
```diff
@@ -26,6 +26,29 @@ You can distinguish it from the rest of the containers by its name:
 
 ![image](/doc_imgs/integrations/66122754-2d337b00-e5e9-11e9-8775-562e228e3fe6.png)
 
+Certain integrations are long-running by default, they do not include the "Long Running Instance" checkbox. Examples of such integrations include:
```
Comment (ShirleyDenkberg) [inline docs/integrations/long-running.md:29]:
```suggestion
Certain integrations are long-running by default so they do not include the "Long Running Instance" checkbox. Examples of such integrations include:
```

File: docs/integrations/long-running.md [hunk 8]
```diff
@@ -26,6 +26,29 @@ You can distinguish it from the rest of the containers by its name:
...
+2. When the engine is selected, a new field called "Listen Port" will appear.
+3. Specify the port number that the instance will use.
+
+**No Engine:**
+1. Select "no engine" in the integration configuration.
+2. Click **Save** and the server automatically assigns a port for the instance.
+
+  Note: If you select **No Engine**, the listen port is assigned only after saving the configuration. If the instance is not saved, the server will assign a temporary port until the  instance starts running. Please make sure to click **Save**, not **Save & Exit**.
+  If you configured the instance but exited before saving, you will need to create a new instance and follow the steps outlined above.
+  In XSOAR 8.9 version, the behavior for "No Engine" will be fixed, and the listen port will be assigned immediately upon configuration, eliminating the need to save the instance manually.
```
Comment (ShirleyDenkberg) [inline docs/integrations/long-running.md:51]:
```suggestion
  In Cortex XSOAR 8.9, the behavior for "No Engine" will change, and the listen port will be assigned immediately upon configuration, eliminating the need to save the instance manually.
```

**Review by Shellyber — APPROVED** [review]

**Pull request merged**